python/src/scipp/_operations.py

```diff
@@ -20,3 +22,29 @@ def sort(x, key, order='ascending'):
     :return: The sorted equivalent of the input.
     """
     return _call_cpp_func(_cpp.sort, x, key, order)
+
+
+def midpoint(low, high):
```

Comment:
Where would you put it instead?

Reply:
Any header that doesn't have all of its contents directly imported in `__init__.py`.

Reply:
Ok. But are you sure that this is not useful to users?

Reply:
It is definitely useful, but maybe not in this form. We'd probably want something like `sc.bin_centers(var)`, so the user does not have to do the slicing. This would be part of a bin-edge toolkit which is currently scattered across many places. We have to sit down at some point and make a decision on naming/API and features. Introducing "random" public functions as a side effect risk adding too many things which we later need to declare as "breaking" changes.